tracer/src/Datadog.Trace/RemoteConfigurationManagement/Transport/RemoteConfigurationFileApi.cs

```diff
@@ -26,10 +26,15 @@ public static RemoteConfigurationFileApi Create(RemoteConfigurationSettings sett
 
         public Task<GetRcmResponse> GetConfigs(GetRcmRequest request)
         {
-            var content = File.ReadAllText(_filePath);
-            var config = JsonConvert.DeserializeObject<GetRcmResponse>(content);
+            if (!File.Exists(_filePath))
+            {
+                return null;
```

Comment:
Did you mean to return a `null` task or `null` response?
```suggestion
                return Task.FromResult(null);
```

Reply:
Nice catch!